static/styles.css

```diff
@@ -21,13 +21,19 @@ Kanban boards styles
     color: white ! important;
 }
 
+.partially-accepted {
+    background-color: #70A3CC !important;
```

Comment:
Inconsistent code styling

should there be a gap between '!' and 'important' like line 21

Reply:
Updated - thanks for letting me know :) 

I'm sure there are other improvements to the code that could be done.

PS. Why hasn't anyone replied to their WEHI emails this week? I think we need to ensure that we can work independently from you (as you can't work in the day) but also together, if you know what I mean....